wagtail/users/templates/wagtailusers/users/create.html

```diff
@@ -13,55 +13,51 @@
-                            <li>{% formattedfield form.email %}</li>
-                            <li>{% formattedfield form.first_name %}</li>
-                            <li>{% formattedfield form.last_name %}</li>
-                            {% block extra_fields %}{% endblock extra_fields %}
-                            {% if form.password1 %}
-                                <li>{% formattedfield form.password1 %}</li>
-                            {% endif %}
-                            {% if form.password2 %}
-                                <li>{% formattedfield form.password2 %}</li>
-                            {% endif %}
-                        {% endblock fields %}
+        <div class="tab-content">
+            <section
+                id="tab-account"
+                class="w-tabs__panel"
+                role="tabpanel"
+                hidden
+                aria-labelledby="tab-label-account"
+            >
+                <ul class="fields">
+                    {% block fields %}
+                        {% if form.separate_username_field %}
+                            <li>{% formattedfield form.username_field %}</li>
+                        {% endif %}
+                        <li>{% formattedfield form.email %}</li>
+                        <li>{% formattedfield form.first_name %}</li>
+                        <li>{% formattedfield form.last_name %}</li>
+                        {% block extra_fields %}{% endblock extra_fields %}
+                        {% if form.password1 %}
+                            <li>{% formattedfield form.password1 %}</li>
+                        {% endif %}
+                        {% if form.password2 %}
+                            <li>{% formattedfield form.password2 %}</li>
+                        {% endif %}
+                    {% endblock fields %}
 
-                        <li>
-                            <a href="#tab-roles" data-tab-trigger class="button">
-                                {% trans "Roles" %}
-                                {% icon name="arrow-right" classname="default" %}
-                            </a>
-                        </li>
-                    </ul>
-                </section>
-                <section
-                    id="tab-roles"
-                    class="w-tabs__panel"
-                    role="tabpanel"
-                    hidden
-                    aria-labelledby="tab-label-roles"
-                >
-                    <ul class="fields">
-                        <li>{% formattedfield form.is_superuser %}</li>
-                        <li>{% formattedfield form.groups %}</li>
-                        <li><button class="button">{% trans "Add user" %}</button></li>
-                    </ul>
-                </section>
-            </div>
-        </form>
+                    <li>
+                        <a href="#tab-roles" data-tab-trigger class="button w-flex w-items-center w-w-fit w-mb-8">
```

Comment:
Classic Sage attention to detail

![classic-sage](https://github.com/user-attachments/assets/912e9024-ccd3-4858-9dad-dfea3af5e6ef)